math/minuit/src/TFitter.cxx

```diff
@@ -137,8 +137,12 @@ void TFitter::GetConfidenceIntervals(Int_t n, Int_t ndim, const Double_t *x, Dou
    Double_t c=0;
 
    Double_t *matr = GetCovarianceMatrix();
-   if (!matr)
+   if (!matr){
+      delete [] grad;
```

Comment:
This is true. On the other hand this is more of a development rather than a fix of a coverity bug and I would wait to require those. In any case, new development should not rely on the pointer to be a nullptr after being freed in my opinion...